rhobs/alerting/data_plane/prometheus.gitops_argocd_app_reconcile_alerts.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-gitops-argocd-app-reconcile-alert
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: gitops-argocd-app-reconcile-alert
+    interval: 1m
+    rules:
+    - alert: GitOpsArgoCDAppReconcileAlert
+      annotations:
```

Comment:
I suggest including the query value `{{ $value | humanizePercentage }}` in the summary/description.

Reply:
I've added the query value.